dev_docs/key_concepts/anatomy_of_a_plugin.mdx

```diff
@@ -88,12 +89,38 @@ plugins/
   You don't need to declare a dependency on a plugin if you only wish to access its types.
 </DocCallOut>
 
-### [2] public/index.ts
+### tsconfig.json
+
+If you are developing in typescript (which we recommend), you will need to add a `tsconfig.json` file. Here is an example file that you would use if adding a plugin into the `examples` directory.
+
+```json
+{
+  "extends": "../../tsconfig.json", // Extend kibana/tsconfig.json
+  "compilerOptions": {
+    "outDir": "./target/types"
+  },
+  "include": [
+    "index.ts",
+    "../../typings/**/*",
+    // The following paths are optional, based on whether you have common code,
```

Comment:
Will the docs system render comments in json without marking them as invalid like GH does?

Reply:
Looks like it!

<img width="986" alt="Screen Shot 2021-08-12 at 10 57 41 AM" src="https://user-images.githubusercontent.com/16563603/129219638-82f43e1a-b13b-44af-9f9b-23e28bb66034.png">